VISS-explainer.md

```diff
@@ -0,0 +1,31 @@
+
+VISS version 1 deferred on access control to individual implementations and as mentioned only supported Websockets. Of the thousands of data points potentially available, a client application could subscribe to an allowed subset, and thus receive initial and updated values over time. The second version expanded on the filters available for Websocket and MQTT subscriptions and also allows for individual requests over these protocols and HTTPS of individual signals or if allowed a set of specified signals. For example, issuing a request to the node Vehicle.Drivetrain.Transmission would, if authorized, return all data available for a vehicle's transmission. 
+
+_Notes on implementation and use_
+
+VISS has known implementations by Volvo, BMW, JLR, Renesas, Bosch, Visteon, Geotab, AGL and Mitsubishi Electric to name a few. 
+The underlying data model VSS is seeing much wider adoption, eg within AWS Fleetwise or accompanying ontology ([VSSo](https://www.w3.org/TR/vsso/)) for machine learning, since its uses go far beyond a single API.
+
+A client application for example could reside on the vehicle itself on an on-board logic, telematics or head unit. From there it can authenticate itself against VISS and based on permissions granted for the specific application, can read or even write data to/from the vehicle. Applications may be user facing, providing information to operator or occupants or performing data sampling for analysis.
+
+There are multiple uses for vehicle telematics data. Electric vehicle charging events can be combined with charging infrastructure, route planning and weather conditions to plan trips on a vehicles' behalf and ensure battery is not depleted. Insurance companies can use it, with owner/operator permission and typically in exchange for a discount on premium, to assess driving behavior. Fleet operators and mechanic services can assess vehicles performance and predict maintenance issues before they arise and leave a vehicle stranded. Information about a vehicles' status, location, fuel/charge level, diagnostics, etc can be conveyed to the owner on the head unit/in-vehicle infotainment or a trusted, paired smartphone. 
+
+Several projects gathered under the umbrella [COVESA CVII](https://wiki.covesa.global/display/WIK4/Common+Vehicle+Interface+Initiative+-+Home) are addressing the mentioned use cases and the required connectivity building blocks where VISS is o important component to address needs within the vehicle, cloud, AI/knowledge graph, service and solution level all centered on the common vehicle data model VSS. Various potential architectures are under discussion, especially as COVESA and AutoSAR are seeking to increase collaboration.

```

Comment:
Should it not be "AUTOSAR" rather than "AutoSAR". AUTOSAR themselves always seem to use capital letter only, see https://www.autosar.org/